numpy/distutils/command/build_ext.py

```diff
@@ -119,7 +119,7 @@ def run(self):
         self.compiler.show_customization()
 
         # Setup directory for storing generated extra DLL files on Windows
-        self.extra_dll_dir = os.path.join(self.build_temp, '.libs')
+        self.extra_dll_dir = os.path.join(self.build_temp, 'core')
```

Comment:
This changes from a generic solution to something that's specific to `numpy` itself. I think we can do require each package to add some code in combination with changes here, but we can't just make numpy-specific directory layout assumptions here.

Reply:
I guess if other packages are using package root level `.libs` folder we'd probably need to retain that default and either have a generic way to offer a customized switch for the DLL storage folder, or bake in a switch if building NumPy only.

Reply:
`scipy-wheels` is still on a commit before https://github.com/numpy/numpy/commit/473a6f067267e752cbfd2c163e95a0387ccb06b5, so it still has `extra-dll` rather than `.libs`. But it looks like we'll need to upgrade that repo to latest numpy master to get the fix for this PATH issue.